android-sdk/src/main/java/com/mobilecoin/lib/FogSeed.java

```diff
@@ -22,11 +22,15 @@
 
     // Bump serial version and read/write code if fields change
     private static final long serialVersionUID = 1L;
+
+    private final long ingestInvocationId;
```

Comment:
I might be totally wrong here, but I think this needs to get serialized in `writeObject` and `readObject`, which would also imply a format change and perhaps a potential issue to existing SDK users?

Reply:
I made the changes to readObject and writeObject. 

I don't think that this change is an issue for existing SDK users because, to my knowledge, the serialization / deserialization feature is not currently being used by any of our clients. In fact, IIRC, this Serialization code is a placeholder for when apps will use it, and we're not even going to be using the Serialization class- instead will use [Parcelable][https://developer.android.com/reference/android/os/Parcelable). 

@voloshyn Can you confirm that this change is backwards compatible and won't break clients?

Reply:
Yes, we are about to remove it in favor of Parcelable, this is not going to break clients